### PR TITLE
Fix race condition where a listener gets a message before the lock is created.

### PR DESCRIPTION
**Pull request by bdraco**
Reported to Home Assistant here
https://github.com/home-assistant/core/issues/35832

I was able to replicate with ~200 restarts.
Fixes
```
Exception in thread zeroconf-Engine:
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/threading.py", line 926, in _bootstrap_inner
    self.run()
  File "/usr/local/lib/python3.7/site-packages/zeroconf/__init__.py", line 1228, in run
    reader.handle_read(socket_)
  File "/usr/local/lib/python3.7/site-packages/zeroconf/__init__.py", line 1301, in handle_read
    self.zc.handle_response(msg)
  File "/usr/local/lib/python3.7/site-packages/zeroconf/__init__.py", line 2501, in handle_response
    with self._handlers_lock:
AttributeError: 'Zeroconf' object has no attribute '_handlers_lock'
```

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
     readme = f.read()
 
 version = (
-    [l for l in open(join(PROJECT_ROOT, 'zeroconf', '__init__.py')) if '__version__' in l][0]
+    [ln for ln in open(join(PROJECT_ROOT, 'zeroconf', '__init__.py')) if '__version__' in ln][0]
```
Comment (jstasiak) [inline setup.py:12]: This doesn't belong in this pull request I think.

Comment (bdraco) [reply setup.py:12]: The CI was failing until I fixed this since `l` is apparently no longer an allowed var name.

I'll send another PR to fix this and then these two should merge cleanly on top

Comment (bdraco) [reply setup.py:12]: Opened #252 which should allow this to merge cleanly on top if I did it right.

Comment (jstasiak) [reply setup.py:12]: Oh, I missed that, cheers.
